sdks/python/tests/unit/evaluation/metrics/llm_judges/answer_relevance/test_metric.py

```diff
@@ -0,0 +1,15 @@
+from opik import logging_messages
+import pytest
+from opik.evaluation.metrics.llm_judges.answer_relevance.metric import AnswerRelevance
+from opik.exceptions import MetricComputationError
+
+
+def test_answer_relevance_score_out_of_range():
+    metric = AnswerRelevance()
+    invalid_model_output = '{"answer_relevance_score": -0.5, "reason": "Score below valid range."}'  # Score < 0.0
+
+    with pytest.raises(
+        MetricComputationError,
+        match=logging_messages.ANSWER_RELEVANCE_SCORE_CALC_FAILED,
+    ):
+        metric._parse_model_output(invalid_model_output)
```

Comment:
`_parse_model_output` is a protected method, we shouldn't access it directly.
If this function became a separate, testable unit of work, let's extract it to a separate module inside answer_relevance namespace (e.g. `parsers.py` with `parse_model_output` function) and import this module to `metric.py`.
And then we can test this module separately.

As a rule of thumb: we should test the objects via their public API, if it's not convenient and you need to access private attributes/functions/methods, it probably means that the logic is not separated good enough and should be refactored.

Reply:
Good comment Sasha! 🙏 

Appied your suggestion 📜 . Exported every metric.py's _parse_model_output to parse_model_output in `parser.py`.

Also, refactored the name to `test_parser.py` instead of `test_metric.py`, as we test the `parsing` of an output, and not the metric's generation of an output.